Gemfile

```diff
@@ -25,6 +25,8 @@ gem "jbuilder", "~> 2.7"
 # Use Active Storage variant
 # gem 'image_processing', '~> 1.2'
 
+# Bootstrap
+gem "bootstrap"
```

Comment:
why do we need the gem? if you have it in package.json that should be enough

see this as an example https://github.com/failure-driven/remote-button/commit/529b840c583e6f0a3850286a0bcd8a729fe14599

Reply:
removed 👍